source/GameInterface/Serialization/Impl/ItemModifierBinaryPackage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Common.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+
+namespace GameInterface.Serialization.Impl
+{
+    [Serializable]
+    public class ItemModifierBinaryPackage : BinaryPackageBase<ItemModifier>
+    {
+        public ItemModifierBinaryPackage(ItemModifier obj, BinaryPackageFactory binaryPackageFactory) : base(obj, binaryPackageFactory)
+        {
+        }
+
+        public override void Pack()
+        {
+            foreach (FieldInfo field in ObjectType.GetAllInstanceFields())
```

Comment:
Exclude any fields with the `CachedData` attribute. This can be done by passing the name to `GetAllInstanceFields` in an IEnumerable